source/common/router/config_impl.cc

```diff
@@ -218,6 +219,12 @@ RouteTracingImpl::RouteTracingImpl(const envoy::api::v2::route::Tracing& tracing
   } else {
     overall_sampling_ = tracing.overall_sampling();
   }
+  for (const auto& tag : tracing.custom_tags()) {
+    Tracing::CustomTagConstSharedPtr ptr = Tracing::HttpTracerUtility::createCustomTag(tag);
+    if (ptr) {
```

Comment:
Per my other comment this should always return a non-null value so this can be removed and the statement inlined imto the emplace.